scripts/CSSRE/lib/py/kafka.py

```diff
@@ -0,0 +1,86 @@
+import openshift as oc
+import copy
+
+def get_kafka_cluster(namespace):
+  with oc.project(namespace), oc.timeout(5*60):
```

Comment:
`refactor`: I would start creating a file `lib/py/settings.py` and put this value as a constant so it can be reused and become more readable.
```
OC_TIMEOUT_5_MINUTES = 5 * 60
```

Reply:
+1